### ISSUE TITLE
gnupg (2.1) tells me it is a development version

### ISSUE THREAD

**Issue by matthiasbeyer** [NixOS/nixpkgs#11899]
```
$ gpg2 --version
gpg (GnuPG) 2.1.10-unknown
libgcrypt 1.6.4
NOTE: THIS IS A DEVELOPMENT VERSION!
It is only intended for test purposes and should NOT be
used in a production environment or with production keys!
Copyright (C) 2015 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Home: ~/.gnupg
Supported algorithms:
Pubkey: RSA, ELG, DSA, ECDH, ECDSA, EDDSA
Cipher: IDEA, 3DES, CAST5, BLOWFISH, AES, AES192, AES256, TWOFISH,
        CAMELLIA128, CAMELLIA192, CAMELLIA256
Hash: SHA1, RIPEMD160, SHA256, SHA384, SHA512, SHA224
Compression: Uncompressed, ZIP, ZLIB, BZIP2
```

This happens since months, so I assume this isn't upstreams fault but how we build gpg.

Maintainer ping: @wkennington

**Comment by vcunat** [comment]
It seems to me that `2.1.*` are "preview" releases https://en.wikipedia.org/wiki/GNU_Privacy_Guard

**Comment by matthiasbeyer** [comment]
So you think best would be to downgrade my gnupg installation to `2.0.*`?

**Comment by vcunat** [comment]
I don't really use it, so it would be better to wait for an opinion of someone else as well. Maybe 2.1 isn't considered too unstable.

**Comment by vcunat** [comment]
I suspect there might've been a (closed) thread about this already.

**Comment by peti** [comment]
Nixpkgs was inconsistent. The attribute `gnupg` referred to version 2.0.x, but `nix-env -i gnupg` would install version 2.1.x. Fixed in d9ad002b2c49fed8fd416cbbf6eba4962e17fa1e.
